operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/BulkDataImportUtil.java

```diff
@@ -244,11 +254,23 @@ public static StorageDetail retrieveStorageDetails(Parameters parameters) throws
         throw buildExceptionWithIssue("$import required 'storageDetail' is not found", IssueType.INVALID);
     }
 
-    private static void checkValidContentEncoding(String contentEncoding) throws FHIROperationException {
+    private void checkValidContentEncoding(String contentEncoding) throws FHIROperationException {
         if (!BulkDataConstants.STORAGE_CONTENT_ENCODING.contains(contentEncoding)) {
             throw buildExceptionWithIssue(
                     "$import invalid 'contentEncoding' for storageDetail for '" + contentEncoding + "'",
                     IssueType.INVALID);
         }
     }
+
+    public FHIROperationException buildExceptionWithIssue(String msg, IssueType issueType)
+            throws FHIROperationException {
+        OperationOutcome.Issue ooi = FHIRUtil.buildOperationOutcomeIssue(msg, issueType);
+        return new FHIROperationException(msg).withIssue(ooi);
+    }
+
+    public FHIROperationException buildExceptionWithIssue(String msg, Throwable cause, IssueType issueType)
+            throws FHIROperationException {
+        OperationOutcome.Issue ooi = FHIRUtil.buildOperationOutcomeIssue(msg, issueType);
+        return new FHIROperationException(msg, cause).withIssue(ooi);
+    }
```

Comment:
should these be private?

also, I wonder if we have these in enough places to promote them to FHIRUtil...but I'd save that for some other time

Reply:
I just moved these existing ones around, and yes, they probably should be in a util, but... I think in this case it'd go to FHIROperationUtil since it throws FHIROperationException or we could generalize it.

Reply:
I can do that in this one, or leave as-is

Reply:
i see it now.  used to be at the top of the class (lines 38 and 44 respectively).  lets call this one good.